upstairs/src/volume.rs

```diff
@@ -457,6 +457,18 @@ impl BlockIO for Volume {
         Ok(())
     }
 
+    async fn deactivate(&self) -> Result<BlockReqWaiter, CrucibleError> {
+        for sub_volume in &self.sub_volumes {
```

Comment:
We wait here on each subvolume and ROP, and only return from this method when all subvolumes and ROPs have returned, right?

Reply:
Correct